history-of-the-quakers/original/19-book-2-chapter-10.adoc

```diff
@@ -146,16 +165,19 @@ in men by the possession of power,
 according to the concurrent accounts historians have
 given us of the independents and their principles:
 
-"`The fanaticism of the independents abolished all ecclesiastical government,
+[.embedded-content-document]
+--
+
+The fanaticism of the independents abolished all ecclesiastical government,
 disdained all creeds and systems, and rejected every ceremony.
 Of all christian sects this was the first, which,
 during its prosperity as well as its adversity,
 always adopted the principle of toleration;
 and it is remarkable that so reasonable a doctrine owed its origin, not to reasoning,
-but to the height of extravagance and fanaticism.`"^
+but to the height of extravagance and fanaticism.^
 footnote:[Hume]
 
-"`The doctrine of toleration, in these enlightened ages,
+The doctrine of toleration, in these enlightened ages,
```

Comment:
whoa, and this is a bit tough to see, but in looking at this, I think these are two separate quotes from two separate books/authors -- so I think around line 179 we should close the `[quote]` and then immediately start another one.